http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Constants.java

```diff
@@ -175,6 +180,14 @@
 
     public static final String ACCEPT_ENCODING = "Accept-Encoding";
 
+    public static final String LOCATION = "Location";
+
+    public static final String HTTP_GET_METHOD = "GET";
+
+    public static final String HTTP_POST_METHOD = "POST";
+
+    public static final String HTTP_HEAD_METHOD = "HEAD";
```

Comment:
I think there is no need for space between the above lines as they are associated together. I am talking about the verbs.

Reply:
Fixed this.